CHANGELOG.md

```diff
@@ -41,32 +41,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - 
 
 ### Changed
-- 
```

Comment:
Why have you removed empty lines? Wee keep them to make probability of the conflict on `CHANGELOG.md` smaller.